airbyte-integrations/connectors/source-airtable/source_airtable/schema_helpers.py

```diff
@@ -103,7 +106,11 @@ def get_json_schema(table: Dict[str, Any]) -> Dict[str, str]:
                 # Other edge cases, if `field_type` not in SIMPLE_AIRTABLE_TYPES, fall back to "simpleText" == `string`
                 # reference issue: https://github.com/airbytehq/oncall/issues/1432#issuecomment-1412743120
                 if complex_type == SchemaTypes.array_with_any:
-                    complex_type["items"] = deepcopy(SIMPLE_AIRTABLE_TYPES.get(field_type))
+                    if field_type in SIMPLE_AIRTABLE_TYPES:
```

Comment:
It's preferable to use the `deepcopy` to avoid cross-referencing, I was getting them before.
Why not use this: 
```python
if field_type in SIMPLE_AIRTABLE_TYPES:
    complex_type["items"] = deepcopy(SIMPLE_AIRTABLE_TYPES.get(field_type, SchemaTypes.string))
else:
    logger.warning(f"Unknown field type: {field_type}, falling back to `simpleText` type")
```